applications/asset_tracker/src/cloud_codec/cloud_codec.h

```diff
@@ -186,6 +201,17 @@ int cloud_encode_light_sensor_data(const struct light_sensor_data *sensor_data,
 				   struct cloud_msg *output);
 #endif /* CONFIG_LIGHT_SENSOR */
 
+
+/**
+ * @brief Checks if data should be sent to the cloud based on config.
```

Comment:
Should or could?

Reply:
`should`  but in the context of the fuction name `could` is probably best.

Reply:
You're the native English speaker in the room, so you decide :)